modules/mobile/app/services/mobile/v0/disability_rating/proxy.rb

```diff
@@ -11,26 +11,35 @@ def initialize(user)
         end
 
         def get_disability_ratings
-          response = rating_service.rating.find_rating_data(@user.ssn)
-          handle_errors!(response)
-          Mobile::V0::Adapters::Rating.new.disability_ratings(response)
+          combine_response = common_service.get_rating_info
+          individual_response = compensation_service.get_rated_disabilities
```

Comment:
Looks like this request is going to wait for the first one to complete. Could they be called in parallel like appointments and claims? (This could also follow in a separate PR)

Reply:
Updated